module-4/backend/week_two.md

```diff
@@ -5,18 +5,59 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+  Syntatical changes along with additions of class's to simplify and give the ability to become an OOPL 
+  
 2. What's the difference between asynchronous and synchronous JavaScript? 
+  Synchronous is the execution of a set of events or functions in a set order where the next executable set of commands will not execute until the previous executes.
+  Asynch give JS the ability to use browser api's or node asynch libraries to simultaneously execute sets of code with one in the JS engine and then the other browser api or libraries but only allowing the asynch/browser api set of code to finish finish when the Engine's call stack is complete. 
+  
 3. What are the four pillars of Object Oriented programming?
+  Abstraction
+  Polymorphism
+  Inheritance
+  Encapsulation
+  
 4. What are some tools available in JavaScript to help you write object oriented code?
+  Class's, constructors, prototypes, this, objects
+  
 5. What are some key concepts to be aware of when refactoring your JavaScript?
+  four pillars of OOP
+  clean/concise
+  understandable
+  asynch functions
+  callbacks
+  tests
+  
 6. What's a callback function and what are some reasons when we use/need callback functions?
+  Callback functions all for the execution/responsibility of another set of code being packaged and transferred waiting to be exectued by the current function. 
+  Ill work on this answer.
+  
 7. What's the scope of variables in Javascript?
+  Scopes of variables are abstract definition of availability of a specific variable.
+  
 8. What's the difference between `==` and `===` in JavaScript?
+  Triple ='s is a very specific comparative in JS that if I remember correctly will compare the modified times of certain compairing objects. 
+  Double ='s Less specific usually a very intuitive comparitive for JS that checks the initial face value of objects being compared.
```

Comment:
what do each of the operators actually check? i encourage you to brush up on this answer!